style.css

```diff
@@ -7,6 +7,21 @@
 	overflow: auto;
 }
 
+.shcb-language {
+	border: 0;
+	clip: rect(1px, 1px, 1px, 1px);
+	-webkit-clip-path: inset(50%);
+	clip-path: inset(50%);
+	height: 1px;
+	margin: -1px;
+	overflow: hidden;
+	padding: 0;
+	position: absolute;
+	width: 1px;
+	word-wrap: normal;
+	word-break: normal;
+}
```

Comment:
This is adapted from the ubiquitous `screen-reader-text` styles used in WordPress.